examples/snippets/interleaved-tables/README.md

```diff
@@ -32,62 +31,55 @@ CREATE TABLE singers (
 ) PRIMARY KEY (singerid);
 
 CREATE TABLE albums (
-    albumid INT64 NOT NULL,
     singerid INT64 NOT NULL,
+    albumid INT64 NOT NULL,
```

Comment:
why do we need to move `albumid`?

Reply:
The order in which the columns are generated by Ruby on Rails 7.1 is different from how they were generated by the `composite-pk` third-party gem that we used before. I updated the order in the README as well so it corresponds 100% with the actual table that is created.